three_ANA_sites/siteInfo.csv

```diff
@@ -1,10 +1,10 @@
-station,station_name,latitude,longitude,drainage_area,constituent,constit_units
-RONC02800,Ronc,1,1,100,NO3,mg L^-1
-RONC02800,Ronc,1,1,100,PT,mg L^-1
-RONC02800,Ronc,1,1,100,Q,cms
-MOGU02900,Mogu,2,2,200,NO3,mg L^-1
-MOGU02900,Mogu,2,2,200,PT,mg L^-1
-MOGU02900,Mogu,2,2,200,Q,cms
-ORIZ02900,Oriz,3,3,300,NO3,mg L^-1
-ORIZ02900,Oriz,3,3,300,PT,mg L^-1
-ORIZ02900,Oriz,3,3,300,Q,cms
\ No newline at end of file
+file,site.id,site.name,lat,lon,basin.area,constituent,units
```

Comment:
(didn't see your last two comments until i had already commented) 
sure, i'm good with the two-file option. there are still compromises involved, but yeah, it eases some of the issues. want to give it a try in your next PR?